app/app_stack.py

```diff
@@ -28,24 +28,15 @@ def __init__(self, scope: Construct, id: str, **kwargs) -> None:
             string_parameter_name="/odin/cdsapi/url",
         )
 
-        download_era5 = _lambda.Function(
+        download_era5 = _lambda.DockerImageFunction(
             self,
             "downloadERA5",
             timeout=Duration.minutes(15),
-            runtime=_lambda.Runtime.PYTHON_3_10,
-            code=_lambda.Code.from_asset(
-                "app/download",
-                bundling={
-                    "image": _lambda.Runtime.PYTHON_3_10.bundling_image,
-                    "command": [
-                        "bash",
-                        "-c",
-                        "pip install -r requirements.txt -t /asset-output && cp -au . /asset-output",
-                    ],
-                },
+            code=_lambda.DockerImageCode.from_image_asset(
```

Comment:
It would be really nice to name (and maybe tag) these images. The ECS-registry gets very messy with just hashed names. Very difficult knowing what might be running where.